src/test/java/org/sonar/plugins/checkstyle/metadata/CheckstyleMetadataTest.java

```diff
@@ -116,24 +121,12 @@ public void testCreate() {
         });
     }
 
-    private ModuleDetails loadMeta(String checkName) {
-        final ModuleDetails moduleDetails;
-        try (InputStream inputStream = getClass().getResourceAsStream(checkName + ".xml")) {
-            moduleDetails = new XMLReader().read(inputStream, ModuleType.CHECK);
-        }
-        catch (IOException ex) {
-            throw new IllegalStateException("exception occured during loadMeta of " + checkName,
-                    ex);
-        }
-        return moduleDetails;
-    }
-
-    private static String convertName(String name) {
+    private static String convertName(String checkName) {
         final int capacity = 1024;
         final StringBuilder result = new StringBuilder(capacity);
-        for (int i = 0; i < name.length(); i++) {
-            result.append(name.charAt(i));
-            if (i + 1 < name.length() && Character.isUpperCase(name.charAt(i + 1))) {
+        for (int i = 0; i < checkName.length(); i++) {
+            result.append(checkName.charAt(i));
+            if (i + 1 < checkName.length() && Character.isUpperCase(checkName.charAt(i + 1))) {
```

Comment:
please move this update to separate commit "minor: change name of argument in convetName method."

Reply:
Done in new reorganized PR https://github.com/checkstyle/sonar-checkstyle/pull/306